weblate/addons/models.py

```diff
@@ -81,7 +89,6 @@ def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
         cls = self.addon_class
-        self.project_scope = cls.project_scope
```

Comment:
It should set `project` instead of `component` in this case (similar like changing the component for the repo_scope below).